### PR TITLE
open-cluster-management/backlog#10581 Add ki bm 47

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -138,6 +138,12 @@ After the `_finalizers_` parameter is removed, the `ObervabilityAddon` resource
 [#cluster-management-issues]
 == Cluster management known issues
 
+[#no-create-bm-47]
+=== Cannot create bare metal managed clusters on {ocp-short} version 4.7
+// 2.2:10581
+
+You cannot create bare metal managed clusters by using the Red Hat Advanced Cluster Management hub cluster when the hub cluster is hosted on {ocp-short} version 4.7.
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: Replace Red Hat Advanced Cluster Management with the conref {product-title-short}

**Review by dockerymick — APPROVED** [review]
One small change, besides that this looks good. thanks for adding it

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues.adoc]: You got it! Thanks!

**Comment by cadawson** [comment]
Approved by Chris Doan in a slack message.

**Pull request merged**